.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+## 📌 Ticket
+
+Please provide the URL to the Jira ticket related to this PR.
+
```

Comment:
Is this necessary? The ticket is usually available in the right column under "Development": 
![image](https://github.com/user-attachments/assets/636ddb8c-b471-4b27-8e9f-6c83c5cb142b)